environment_derived_columns/templates/db_system_or_type.tftpl

```diff
@@ -0,0 +1,5 @@
+COALESCE(
+  IF(EXISTS($db.system), $db.system),
```

Comment:
I don't think you need the `IF(EXISTS()` here, because COALESCE is designed to deal with missing/null columns right?

Reply:
You're correct! I didn't.